src/places.js

```diff
@@ -2,6 +2,19 @@ module.exports = createPlacesClient;
 
 var buildSearchMethod = require('./buildSearchMethod.js');
 
+function urlEncodeOptions(options) {
```

Comment:
Not sure if relevant but we already use [`querystring-es3`](https://github.com/mike-spainhower/querystring) in the project which should be able to replace this custom function.

Reply:
Thanks for the info! I updated the code to use the library instead. 👍